requirements.txt

```diff
@@ -11,4 +11,5 @@ djangorestframework-gis==0.11
 dj-database-url==0.4.2
 gunicorn==19.6.0
 whitenoise==3.3.0
-pandas==0.20.1
\ No newline at end of file
+pandas==0.20.1
+numpy==1.12.1
```

Comment:
I'm happy with this as long as we consider it a temporary workaround and not a fix, otherwise we are binding us to an old/deprecated version of Numpy. I'm going to create a ticket for upgrade the code so that it's compatible with later releases of this library.